### PR TITLE
Implement Extend for WriteHandle

### PR DESCRIPTION
**Pull request by glittershark**
Add a impl for Extend<Op> to WriteHandle, to append multiple operations
from an iterator from the oplog at once. In addition to convenience,
this has the potential to be more efficient by avoiding reallocating the
vector for the oplog multiple times.

---
This change is [Reviewable](https://reviewable.io/reviews/jonhoo/rust-evmap/86)

### CODE REVIEW

**Comment by jonhoo** [comment]
That's a great idea, love it! One little nit inline and then I'm happy to merge this.

File: left-right/src/write.rs [hunk 1]
```diff
@@ -299,18 +299,7 @@ where
...
-            let mut w_inner = self.raw_write_handle();
-            let w_inner = unsafe { w_inner.as_mut() };
-            let r_handle = self.enter().expect("map has not yet been destroyed");
-            // Because we are operating directly on the map, and nothing is aliased, we do want
-            // to perform drops, so we invoke absorb_second.
-            Absorb::absorb_second(w_inner, op, &*r_handle);
-        } else {
-            self.oplog.push_back(op);
-        }
+        self.extend(vec![op]);
```
Comment (jonhoo) [inline left-right/src/write.rs:302]: I think this should probably use `std::iter::once` so we don't have to allocate a `Vec`.

Comment (glittershark) [reply left-right/src/write.rs:302]: ah, yes, I knew something like that had to exist!

**Review by jonhoo — APPROVED** [review]

**Comment by jonhoo** [comment]
Thanks!

**Comment by jonhoo** [comment]
Published in `left-right 0.11` and `evmap 11.0.0-alpha.5` :tada:

**Pull request merged**